CODEOWNERS

```diff
@@ -62,5 +62,5 @@
 /scripts/delete_failing_smt2_solver_tests
 /scripts/expected_doxygen_warnings.txt
 
-/.travis.yml @diffblue/devops @thk123 @forejtv @peterschrammel
+/.travis.yml @diffblue/devops @thk123 @forejtv @peterschrammel @chrisr-diffblue
 /appveyor.yml @diffblue/devops @thk123 @forejtv @peterschrammel
```

Comment:
Would you mind removing that `appveyor.yml` line (possibly in a separate commit)?

Reply:
Sure!